controller_manager:
  ros__parameters:
    update_rate: 100

joint_state_broadcaster:
  ros__parameters:
    type: joint_state_broadcaster/JointStateBroadcaster
    publish_rate: 50

arm_controller:
  ros__parameters:
    type: joint_trajectory_controller/JointTrajectoryController
    joints:
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5
      - joint_6
      - joint_7
    state_publish_rate: 50
    action_monitor_rate: 20
    constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.02
      joints:
        joint_1: { trajectory: 1.0, goal: 0.5 }
        joint_2: { trajectory: 1.0, goal: 0.5 }
        joint_3: { trajectory: 1.0, goal: 0.5 }
        joint_4: { trajectory: 1.0, goal: 0.5 }
        joint_5: { trajectory: 1.0, goal: 0.5 }
        joint_6: { trajectory: 1.0, goal: 0.5 }
        joint_7: { trajectory: 1.0, goal: 0.5 }
